server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/reader/CobolLineReaderImpl.java

```diff
@@ -150,8 +151,17 @@ public CobolLineReaderImpl(CobolLineReaderDelegate delegate, MessageService mess
                             INDICATOR_AREA_INDEX + 1))));
   }
 
+  /**
+   *
+   * @param uri the document URI
+   * @param message the error message
+   * @param lineNumber the error lineNumber
+   * @param start the error start position
+   * @param stop the error stop position
+   * @return the {@link SyntaxError}
+   */
   @NonNull
-  private SyntaxError createError(
+  public static SyntaxError createError(
```

Comment:
Why is it public?

Reply:
I agree. It was me being lazy. Corrected.